webshare/share-securecontext.http.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
It appears the automated testing above is correctly running share-securecontext.http.html on a non-secure origin so I think this is working out.